diff --git a/Makefile.in b/Makefile.in
index b430efb..ef93598 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -32,6 +32,7 @@ examplesdir = $(datadir)/@PACKAGE_NAME@/examples
 # install.                                                                     #
 ################################################################################
 CFLAGS       = @CFLAGS@ -Wall -DVERSION=\"@PACKAGE_VERSION@\" -Dsysconfdir=\"$(sysconfdir)\" @FFMPEG_CFLAGS@
+CPPFLAGS     = @CPPFLAGS@
 LDFLAGS      = @LDFLAGS@
 LIBS         = @LIBS@  @FFMPEG_LIBS@
 VIDEO_OBJ    = @VIDEO@
@@ -119,7 +120,7 @@ endif
 ################################################################################
 $(DEPEND_FILE): *.h $(SRC)
 	@echo "Generating dependencies, please wait..."
-	@$(CC) $(CFLAGS) -M $(SRC) > .tmp
+	@$(CC) $(CFLAGS) $(CPPFLAGS) -M $(SRC) > .tmp
 	@mv -f .tmp $(DEPEND_FILE)
 	@echo
 
