diff --git a/auto/lib/openssl/make b/auto/lib/openssl/make
index a6090c6..765cd06 100644
--- a/auto/lib/openssl/make
+++ b/auto/lib/openssl/make
@@ -53,7 +53,7 @@ $OPENSSL/.openssl/include/openssl/ssl.h:	$NGX_MAKEFILE
 	&& if [ -f Makefile ]; then \$(MAKE) clean; fi \\
 	&& ./config --prefix=$ngx_prefix no-shared $OPENSSL_OPT \\
 	&& \$(MAKE) \\
-	&& \$(MAKE) install_sw LIBDIR=lib
+	&& \$(MAKE) install LIBDIR=lib
 
 END
 
